### ISSUE TITLE
Could you provide the trainning config you used for standard dataset (standard/re2com.yaml)?

### ISSUE THREAD

**Issue by wuchiuwong** [Gompyn/re2com-opensource#2]
When I tried to reproduce the experiment on the standard data set, the BLEU of the trained model was only 23.2, which is a certain gap from the 24.42 recorded in the paper.
After comparing the training configs (<code>standard/re2com.yaml</code>), I found that some of the settings are different from the description in the paper, such as:
In paper:
> we set the batch size to 256

In <code>standard/re2com.yaml</code>:

> batch_size: 2 #56         # batch size (during training and greedy decoding)

(I have modified it to 256 )

In paper:
> To mitigate overfitting, we use dropout with 0.2

In <code>standard/re2com.yaml</code>:

> pervasive_dropout: False  # same dropout mask for all elements in same batch/same sequence (see Gal, 2015)
> rnn_input_dropout: 0.0
> rnn_output_dropout: 0.0
> rnn_state_dropout: 0.0
> initial_state_dropout: 0.0
> word_dropout: 0.0
> input_layer_dropout: 0.0
> output_dropout: 0.0      # TODO
> use_dropout: False

The building result of exemplars has been checked, and the BLEU of the exemplars in the test set is 18.04, which is the same as the paper. My guess is that the difference in parameter configuration causes the difference in experimental results. Can you provide the correct training settings ?

**Comment by Bolin0215** [comment]
I'm very sorry that my original config file is on a broken server.
I modified the config file according to the settings in the paper.
Please try again to see if you can get better results.

**Comment by Bolin0215** [comment]
> I'm very sorry that my original config file is on a broken server.
> I modified the config file according to the settings in the paper.
> Please try again to see if you can get better results.

**Comment by Bolin0215** [comment]
> When I tried to reproduce the experiment on the standard data set, the BLEU of the trained model was only 23.2, which is a certain gap from the 24.42 recorded in the paper.
> After comparing the training configs (`standard/re2com.yaml`), I found that some of the settings are different from the description in the paper, such as:
> In paper:
> 
> > we set the batch size to 256
> 
> In `standard/re2com.yaml`:
> 
> > batch_size: 2 #56         # batch size (during training and greedy decoding)
> 
> (I have modified it to 256 )
> 
> In paper:
> 
> > To mitigate overfitting, we use dropout with 0.2
> 
> In `standard/re2com.yaml`:
> 
> > pervasive_dropout: False  # same dropout mask for all elements in same batch/same sequence (see Gal, 2015)
> > rnn_input_dropout: 0.0
> > rnn_output_dropout: 0.0
> > rnn_state_dropout: 0.0
> > initial_state_dropout: 0.0
> > word_dropout: 0.0
> > input_layer_dropout: 0.0
> > output_dropout: 0.0      # TODO
> > use_dropout: False
> 
> The building result of exemplars has been checked, and the BLEU of the exemplars in the test set is 18.04, which is the same as the paper. My guess is that the difference in parameter configuration causes the difference in experimental results. Can you provide the correct training settings ?

I'm very sorry that my original config file is on a broken server.
I modified the config file according to the settings in the paper.
Please try again to see if you can get better results.
